option optimize_for = LITE_RUNTIME;

message ChipMergeRequest
{
 required uint32 id = 1;
 required uint32 auto_merge = 2;
}
